src/main/java/net/sf/jabref/logic/msbib/MSBibMapping.java

```diff
@@ -103,19 +111,32 @@ public static MSBibEntryType getMSBibEntryType(String bibtexType) {
         entryTypeMapping.put("unpublished", MSBibEntryType.Report);
         entryTypeMapping.put("patent", MSBibEntryType.Patent);
         entryTypeMapping.put("misc", MSBibEntryType.Misc);
-        entryTypeMapping.put("electronic", MSBibEntryType.Misc);
+        entryTypeMapping.put("electronic", MSBibEntryType.ElectronicSource);
+        entryTypeMapping.put("online", MSBibEntryType.InternetSite);
 
         return entryTypeMapping.getOrDefault(bibtexType, MSBibEntryType.Misc);
     }
 
     // http://www.microsoft.com/globaldev/reference/lcid-all.mspx
+    /**
+     *  Only English is supported
+     * @param language
+     * @return Returns 0 for English
+     */
+    @Deprecated
     public static int getLCID(String language) {
         // TODO: add language to LCID mapping
         // 0 is English
         return 0;
     }
 
     // http://www.microsoft.com/globaldev/reference/lcid-all.mspx
```

Comment:
Isn't `@Deprecated` for methods that eventually should be removed, rather than for methods that are still not properly implemented? I think that with increased BibLatex support we will add support for `language` and `lang-id`. If they are in any way related to the lcid I have no idea about though...